source/hardware/controllers/native_instruments_traktor_kontrol_s4_mk3.rst

```diff
@@ -0,0 +1,404 @@
+|                  | - While in sample:                                               |   note, if Dim on, currently             |
+|                  |                                                                  |   active note                            |
+|                  |   - press will play (load selected track if none are)            | - In Beatloop roll: dimed on means a     |
+|                  |   - :hwlabel:`SHIFT` + press will stop (if playing) or eject     |   loop rool is active with the given     |
+|                  |                                                                  |   size                                   |
+|                  | - While in keyboard:                                             |                                          |
+|                  |                                                                  |                                          |
+|                  |   - will set the key to the selected note                        |                                          |
+|                  |   - will play from the cue if in keyboard play mode              |                                          |
+|                  |                                                                  |                                          |
+|                  | - While in beatlooproll mode:                                    |                                          |
+|                  |                                                                  |                                          |
+|                  |   - will activate a beatlooproll of 0.065, 0.125, 0.25. 0.5, 1,  |                                          |
+|                  |     2, 4 or 8 beats                                              |                                          |
++------------------+------------------------------------------------------------------+------------------------------------------+
+| FX 1st           | - Master volume/mix of the unit                                  |                                          |
+| knob             | - Master volume/mix of the unit                                  |                                          |
++------------------+------------------------------------------------------------------+------------------------------------------+
+| FX 1st           | - Trigger all effect                                             | - On if all effect are off and not       |
+| FX 1st           | - Trigger all effect                                             |   pressing :hwlabel:`SHIFT`              |
+| button           | - Assign/de-assign effect to master while pressing               | - On when effect is attached to master   |
+| button           |   :hwlabel:`SHIFT` and no focused effect                         |   and pressing :hwlabel:`SHIFT`          |
```

Comment:
I wonder if `FX 1st button` will be rendered correct - can't check without preview